flowey/flowey_hvlite/src/pipelines/checkin_gates.rs

```diff
@@ -709,6 +710,36 @@ impl IntoPipeline for CheckinGatesCli {
                 );
 
             all_jobs.push(job.finish());
+
+            if arch == CommonArch::X86_64 && !release {
```

Comment:
why `!release`? I would expect this to be a `matches!(config, PipelineConfig::Pr)`, which is a more accurate semantic mapping